src/components/dev-hub/academia-sign-up-form.js

```diff
@@ -0,0 +1,213 @@
+import React, { useState } from 'react';
+import styled from '@emotion/styled';
+import { colorMap, size, screenSize } from './theme';
+import { P, ArticleH3, H3 } from './text';
+import Input from './input';
+import Button from './button';
+import FormSelect from './select';
+import Checkbox from '@leafygreen-ui/checkbox';
+import SectionHeader from './section-header';
+import SuccessIcon from './icons/success';
+
+const StyledButton = styled(Button)`
+    display: flex;
+    margin: ${size.large} auto;
+`;
+
+const StyledInput = styled(Input)`
+    height: 40px;
+    margin-bottom: ${size.large};
+    width: 335px;
+`;
+
+const StyledFormSelect = styled(FormSelect)`
+    color: ${colorMap.greyLightTwo};
+    height: 40px;
+    width: 335px;
+`;
+
+const SuccessContainer = styled('div')`
+    text-align: center;
+    svg {
+        margin-bottom: ${size.large};
+    }
+`;
+
+const ErrorMessage = styled(P)`
+    color: ${colorMap.salmon};
+`;
+
+const StyledSectionText = styled(ArticleH3)`
+    color: ${colorMap.greyLightTwo};
+    font-family: akzidenz;
+    font-weight: normal;
+`;
+
+const InstructorSection = styled('div')`
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    margin-bottom: ${size.default};
+    @media ${screenSize.upToMedium} {
+        display: block;
+    }
+`;
+
+const InstitutionSection = styled('div')`
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: ${size.default};
+    @media ${screenSize.upToMedium} {
+        display: block;
+    }
+`;
+
+const StyledForm = styled('form')`
+    margin: 0 auto;
+    width: 720px;
+`;
+
+const Form = React.memo(({ setSuccess, success }) => {
+    const [firstName, setFirstName] = useState('');
+    const [lastName, setLastName] = useState('');
+    const [email, setEmail] = useState('');
+    const [institutionName, setInstitutionName] = useState('');
+    const [institutionType, setInstitutionType] = useState('');
+    const [agreeToEmail, setAgreeToEmail] = useState(false);
+    const [canSubmit, setCanSubmit] = useState(true);
+    const institutionTypes = [
```

Comment:
Ah can we update the values (not the labels) here to be underscore_case and remove the slashes? I have a feeling that might be easier to work with on the backend.

i.e. "University / College" --> university_college" for the value (not the label)